src/sentry/monitors/endpoints/organization_monitor_schedule_sample_data.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime
+
+from croniter import croniter
+from dateutil import rrule
+from django.utils import timezone
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.models.organization import Organization
+from sentry.monitors.schedule import SCHEDULE_INTERVAL_MAP
+
+
+@region_silo_endpoint
+class OrganizationMonitorScheduleSampleDataEndpoint(OrganizationEndpoint):
```

Comment:
let's make sure this endpoint is set to private and has owners, etc.

Reply:
oooh good call lemme fix